state-chain/chains/src/address.rs

```diff
@@ -175,6 +191,14 @@ impl EncodedAddress {
 				Ok(EncodedAddress::Dot(address))
 			},
 			ForeignChain::Bitcoin => Ok(EncodedAddress::Btc(bytes)),
+			ForeignChain::Arbitrum => {
```

Comment:
another good reason to remove the arb version of encoded address here - removes this duplication

Reply:
update: we actually need the correct address: https://github.com/chainflip-io/chainflip-backend/pull/4486#discussion_r1502401776